src/components/two-up-display.sass

```diff
@@ -64,12 +71,7 @@
           .button
             fill: $two-up-button-icon-color-active
 
-      &.disabled
-        .button
-          opacity: 0.25
-      &.active
-        .button
-          fill: $two-up-button-icon-color-active
+
```

Comment:
You could unindent and make a slight tweak to the previous version to just define the fill once, right? Something like

```
.button-holder
  &.disabled
    .button
      opacity: 0.25
  &.active
    .button
      fill: $two-up-button-icon-color-active
```

Reply:
It's already defined exactly like that a few lines up. The issue is that the organism-specific rule

`.header.organism .button-holder .button`

is higher priority than the generic header + active:

`.header .button-holder .button.active`

and so the top one still applies even when the button is active, so I need to create another rule for 

`.header.organism .button-holder .button.active`

If you can get another way to work that's cleaner, go for it, but I didn't see anything that simplified the number of rules.